packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -65,21 +65,17 @@ const getStateFromProps = (props: UseSendFormProps) => {
     // utxos might be spent using CoinControl feature
     const excludedUtxos: UseSendFormState['excludedUtxos'] = {};
     if (account.utxo) {
-        const coinjoinSession = props.coinjoinAccount?.session;
         const targetAnonymity = props.coinjoinAccount?.targetAnonymity || 1;
         const anonymitySet = account.addresses?.anonymitySet || {};
         account.utxo?.forEach(utxo => {
             const outpoint = getUtxoOutpoint(utxo);
             const anonymity = anonymitySet[utxo.address] || 1;
-            if (coinjoinSession && coinjoinSession.registeredUtxos.includes(outpoint)) {
-                // utxo is registered in coinjoin
-                excludedUtxos[outpoint] = 'mixing';
+            if (new BigNumber(utxo.amount).lte(Number(coinFees.dustLimit))) {
```

Comment:
the change to `lte` is deliberate?

Reply:
Yes, to correspond with the [validation](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/api/signTransaction.ts#L103). It is quite unintuitive, though.